client/blocks/inline-help/popover-content.scss

```diff
@@ -0,0 +1,168 @@
+.inline-help__popover-content {
+	background-color: var( --color-surface );
+
+	p {
+		font-size: $font-body-extra-small;
+
+		@include breakpoint-deprecated( '>660px' ) {
+			font-size: $font-body-small;
+		}
+	}
+
+	a,
+	button {
+		font-size: $font-body-extra-small;
+
+		@include breakpoint-deprecated( '>660px' ) {
+			font-size: $font-body-small;
+		}
+	}
+
+	&.is-secondary-view-active {
+		.inline-help__search {
+			display: none;
+		}
+
+		.inline-help__secondary-view {
+			display: block;
+		}
+
+		.inline-help__footer {
+			.inline-help__more-button,
+			.inline-help__contact-button {
+				display: none;
+			}
+
+			.inline-help__cancel-button {
+				display: block;
+			}
+		}
+	}
+}
+
+.inline-help__secondary-view {
+	display: none;
+
+	box-sizing: border-box;
+	max-height: 70vh;
+	min-height: 206px;
+	overflow-y: auto;
+	padding: 16px;
+
+	.button {
+		width: 100%;
+		text-align: center;
+	}
+}
+
+.inline-help__view-heading {
```

Comment:
This lives only in the `InlineHelpForumView` component. Should we move it to its own stylesheet there?

Reply:
Moved in e48c4fbe49